yew/src/virtual_dom/vtag.rs

```diff
@@ -736,11 +736,10 @@ mod tests {
     #[test]
     fn supports_multiple_classes_string() {
         let a = html! {
-            <div class="class-1 class-2 class-3"></div>
+            <div class=("class-1 class-2 class-3")></div>
```

Comment:
This won't split without parenthesis anymore but both syntax are actually working